src/main/commands/scan.yml

```
@@ -4,6 +4,10 @@ parameters:
     description: the name of the environment variable where the SonarCloud API token is stored 
     default: SONAR_TOKEN
     type: env_var_name
+  sonar_config_path:
```

I think `sonar_properties_path would be a better name for this property.
```suggestion
  sonar_properties_path:
```